.github/workflows/dependabot.yml

```diff
@@ -18,7 +18,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - name: Check out code into the Go module directory
-      uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab # tag=v3.5.2
+      uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # tag=v3.5.2
```

Comment:
Here dependabot doesn't update the tag in comments, leading to confusion.

Reply:
interesting, is it because of the format ? here it is doing it correctly https://github.com/metal3-io/ip-address-manager/pull/257/ and the format is a bit different

Reply:
It also fails for lint-docs.yaml, where the format is same as others.

Reply:
might be related to this https://github.com/dependabot/dependabot-core/issues/7376